# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1452275368 18000
#      Fri Jan 08 12:49:28 2016 -0500
# Node ID 09c7536502f97f5063212c8ee9af6be76554c8b5
# Parent  8190fe6dc3fd116efbb12bc53554e52b7614934f
* data.cc: Fix typo in previous change.

diff --git a/libinterp/corefcn/data.cc b/libinterp/corefcn/data.cc
--- a/libinterp/corefcn/data.cc
+++ b/libinterp/corefcn/data.cc
@@ -1162,17 +1162,16 @@ For example:\n\
 @example\n\
 @group\n\
 cumsum ([1, 2; 3, 4; 5, 6])\n\
    @result{}  1   2\n\
        3   8\n\
       15  48\n\
 @end group\n\
 @end example\n\
-@end example\n\
 @seealso{prod, cumsum}\n\
 @end deftypefn")
 {
   DATA_REDUCTION (cumprod);
 }
 
 /*
 %!assert (cumprod ([1, 2, 3]), [1, 2, 6])
@@ -1209,17 +1208,16 @@ For example:\n\
 @example\n\
 @group\n\
 cumsum ([1, 2; 3, 4; 5, 6])\n\
    @result{}  1   2\n\
        4   6\n\
        9  12\n\
 @end group\n\
 @end example\n\
-@end example\n\
 \n\
 See @code{sum} for an explanation of the optional parameters\n\
 @qcode{\"native\"}, @qcode{\"double\"}, and @qcode{\"extra\"}.\n\
 @seealso{sum, cumprod}\n\
 @end deftypefn")
 {
   int nargin = args.length ();
 
